docs/ChangeLog/20210227.md

```diff
@@ -0,0 +1,118 @@
+# QMK Breaking Changes - 2021 February 27 Changelog
+
+## Changes Requiring User Action
+
+The following keyboards have had their source moved within QMK:
+
+Old Keyboard Name | New Keyboard Name
+:---------------- | :----------------
+bear_65 | jacky_studio/bear_65
+s7_elephant/rev1 | jacky_studio/s7_elephant/rev1
+s7_elephant/rev2 | jacky_studio/s7_elephant/rev2
+aplx6 | aplyard/aplx6/rev1
+southpaw75 | fr4/southpaw75
+
+The [Aplyard Aplx6 rev2](https://github.com/qmk/qmk_firmware/tree/0.12.0/keyboards/aplyard/aplx6/rev1) and the [FR4Boards Unix60](https://github.com/qmk/qmk_firmware/tree/0.12.0/keyboards/fr4/unix60) have also been added as part of these changes.
+
+Additionally, the `handwired/bluepill/bluepill70` keyboard has been removed.
+
+* Refactor Jacky's boards (Bear65 and S7 Elephant) ([#10528](https://github.com/qmk/qmk_firmware/pull/10528), [#11981](https://github.com/qmk/qmk_firmware/pull/11981))
+* Remove handwired/bluepill ([#11415](https://github.com/qmk/qmk_firmware/pull/11415))
+* Aplyard Aplx6 Added rev2 & move rev1+rev2 to parent folder ([#10973](https://github.com/qmk/qmk_firmware/pull/10973))
+* added `unix60`, moved together with `southpaw75` into `fr4` folder ([#11195](https://github.com/qmk/qmk_firmware/pull/11195))
+
+
+## Core Changes
+
+### Fixes
+
+* GCC 10 can now compile Drop Alt firmware ([#9485](https://github.com/qmk/qmk_firmware/pull/9485))
+* Fix compiling on `develop` branch ([#11409](https://github.com/qmk/qmk_firmware/pull/11409))
+* Fix broken keyboards and keymaps ([#11412](https://github.com/qmk/qmk_firmware/pull/11412), [#11427](https://github.com/qmk/qmk_firmware/pull/11427), [#11448](https://github.com/qmk/qmk_firmware/pull/11448), [#11447](https://github.com/qmk/qmk_firmware/pull/11447), [#11473](https://github.com/qmk/qmk_firmware/pull/11473), [#11584](https://github.com/qmk/qmk_firmware/pull/11584), [#11600](https://github.com/qmk/qmk_firmware/pull/11600))
+* Fixes up build dependencies so that the generated files are correctly made available previous to compiling any object files ([#11435](https://github.com/qmk/qmk_firmware/pull/11435))
```

Comment:
```suggestion
* Fixed up build dependencies so that generated files are made available before compiling any object files ([#11435](https://github.com/qmk/qmk_firmware/pull/11435))
```